### ISSUE TITLE
SVG icons in search results should be marked as hidden or provided with alternative text

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#47]
<img width="1275" height="617" alt="Image" src="https://github.com/user-attachments/assets/39024c9c-6211-4dcc-a086-00da7c5005db" />

Image shows the `svg` code inside search results, which is a raw `svg` with no alternative text and not hidden.

**Comment by rianrietveld** [comment]
Fixed in #52 
props @joedolson
